quickstart-html/01 Credits.html

```diff
@@ -283,22 +283,41 @@ <h1>Modiphius Entertainment</h1>
   </div>
 </section>
 
+<section class="system-support">
+  <h2 class="no-toc">System Support / Contact</h2>
+  <img
+    src="systems/discworld-modiphius/assets/metamorphic-logo-large.webp"
+    alt="Metamorphic Digital Studio Logo">
+  <p>
+    <strong>Foundry VTT Team: </strong>
+    This system was prepared under the direction of Dr. Amy Bliss Marshall by the <a
+      href="https://metamorphic-digital.com/">MetaMorphic Digital Studio</a> VTT Conversion Team, including: Zhell, Rudy Basso, and  Jalen Michals Levy.
+  </p>
+  <p>
+    <strong>To report bugs or other issues with this product, please fill out a support request with our service desk:
+    </strong>
+    <a href="https://support.metamorphic-digital.com/">https://support.metamorphic-digital.com/</a>
```

Comment:
think we should include this as just an alternative to the github issue tracker.